scripts/docker/gatkbase/Dockerfile

```diff
@@ -15,23 +15,28 @@ RUN apt-get update && \
     less \
     bedtools \
     samtools \
-    openjdk-8-jdk \
     tabix \
     gpg-agent \
     build-essential \
+    openjdk-17-jdk \
     software-properties-common && \
+	
     apt-get -y clean  && \
     apt-get -y autoclean  && \
     apt-get -y autoremove
 
+RUN java -version
+
 #### Specific for google cloud support
-RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
-    echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
-    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
-    apt-get update -y && apt-get install google-cloud-sdk -y && \
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" \
+    | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg \
+    | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && \
+    apt-get update -y && \
+    apt-get install -y --no-install-recommends google-cloud-cli && \
+    apt-get -y clean  && \
+    apt-get -y autoclean  && \
```

Comment:
I'm so glad someone knows how todo this...

Reply:
I suspect I'm repeating myself because I don't really know how to do it.  Throw all the cleaning at the wall and hope something sticks.